app.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
-const app = require('./');
+const hof = require('hof');
+const config = require('./config');
+const mockAPIs = require('./mock-apis');
+const bodyParser = require('busboy-body-parser');
+const notProd = config.env !== 'production';
+
+const whatCSP = () => {
+  if (notProd) {
+    return false;
+  }
+  return {
+    // required to use data:image for image-preview
+    imgSrc: ['data:']
+  };
+};
+
```

Comment:
Since hof uses the `DISABLE_CSP` environment variable to disable CSP, it would be nice to use the same env var here to avoid piggy-backing everything off the `NODE_ENV`, which makes it really hard to do fine-grained configuration.

Reply:
I think I'm going to fix this in `hof` so that passing custom CSP directives and disabling csp completely are separately configured. That saves having to do this dance externally.

Reply:
Done - this PR should help clean this up a bit. https://github.com/UKHomeOfficeForms/hof/pull/160